dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -637,22 +637,34 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
 
     public List<String> getAllowedFFDCAnnotationFromTest(FrameworkMethod m) {
```

Comment:
minor nit: since this method is only ever called from 1 spot within this test class, lets change visibility to private, and make it return `Set<String>` instead of converting from Set -> List at the end